tests/modules/services/screen-locker/basic-configuration.nix

```diff
@@ -0,0 +1,27 @@
+{ config, pkgs, ... }:
+
+{
+  config = {
+    services.screen-locker = {
+      enable = true;
+      inactiveInterval = 5;
+      lockCmd = "${pkgs.i3lock}/bin/i3lock -n -c AA0000";
+      xss-lock = { extraOptions = [ "-test" ]; };
+      xautolock = {
+        enable = true;
+        detectSleep = true;
+        extraOptions = [ "-test" ];
+      };
+    };
```

Comment:
```suggestion
    services.screen-locker = {
      enable = true;
      inactiveInterval = 5;
      lockCmd = "${pkgs.i3lock}/bin/i3lock -n -c AA0000";
      xss-lock = { extraOptions = [ "-test" ]; };
      xautolock = {
        enable = true;
        detectSleep = true;
        extraOptions = [ "-test" ];
      };
    };
    
    test.stubs.i3lock = { };
    test.stubs.xss-lock = { };
```

We need a stubbed package for i3lock because we don't want to pull the package when running the test suite.

Can you also make this change in the other tests?